z/z-schema/z-schema_rhel_8.4.sh

```diff
@@ -0,0 +1,35 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : z-schema
+# Version       : v4.2.2, v4.2.3 and v5.0.0
+# Source repo   : https://github.com/koolhazz/paping.git
+# Tested on     : RHEL 8.4
+# Script License: Apache License Version 2.0
+# Maintainer    : sachin.kakatkar@ibm.com
+#
```

Comment:
There are now additional fields in the template like Language, please update to include those